src/nvim/options.lua

```diff
@@ -1599,7 +1599,7 @@ return {
       deny_duplicates=true,
       alloced=true,
       varname='p_nf',
-      defaults={if_true={vi="octal,hex", vim="hex"}}
+      defaults={if_true={vi="bin,octal,hex", vim="bin,hex"}}
```

Comment:
> Guess then vi default means vim default and vim default is neovim default, in which case this is completely correct.

This was what I guessed too.  The Vim code only has one set of defaults which are now `bin,octal,hex`.